service/src/main/kotlin/fi/espoo/evaka/pis/service/MergeService.kt

```diff
@@ -115,11 +115,7 @@ class MergeService(private val asyncJobRunner: AsyncJobRunner<AsyncJob>) {
                 """
                     (SELECT count(*) FROM $table WHERE $column = :id)
                 """.trimIndent()
-            }} +
-                (SELECT count(*) FROM message WHERE sender_id = (SELECT id FROM message_account WHERE person_id = :id)) +
-                (SELECT count(*) FROM message_content WHERE author_id = (SELECT id FROM message_account WHERE person_id = :id)) +
-                (SELECT count(*) FROM message_recipients WHERE recipient_id = (SELECT id FROM message_account WHERE person_id = :id)) +
-                (SELECT count(*) FROM message_draft WHERE account_id = (SELECT id FROM message_account WHERE person_id = :id)) AS count;
+            }}
```

Comment:
These are OK. If they exist, they're just left behind when the user is deleted, as `message_account.person_id` is set to NULL when a person is deleted.